src/main/java/tutorlink/commandpackage/GetCourseGradeStudentCommand.java

```diff
@@ -0,0 +1,66 @@
+package tutorlink.commandpackage;
+
+import tutorlink.coursepackage.LetterGrade;
+import tutorlink.exceptionspackage.CourseNotFoundException;
+import tutorlink.exceptionspackage.IllegalValueException;
+import tutorlink.exceptionspackage.InvalidWeightingException;
+import tutorlink.exceptionspackage.StudentNotFoundException;
+import tutorlink.coursepackage.Course;
+import tutorlink.listpackage.CourseList;
+import tutorlink.resultpackage.CommandResult;
+import tutorlink.studentpackage.StudentClass;
+
+public class GetCourseGradeStudentCommand extends Command {
+
+    private String courseID;
+    private String studentName;
+
+    public GetCourseGradeStudentCommand(String courseID, String name) {
+        this.courseID = courseID;
+        this.studentName = name;
+    }
+
+    private StudentClass findStudent() throws StudentNotFoundException {
```

Comment:
Could you use CommandResult FindStudentCommand :: execute() instead to reduce code repetition?